src/envoy/prototype/server_config.pb.txt

```diff
@@ -2,5 +2,5 @@ cloud_tracing_config {
   force_disable: true
 }
 mixer_options {
-  mixer_server: "mixer_server"
-}
\ No newline at end of file
+  mixer_server: "localhost:9091"
```

Comment:
Yes, we don't need cluster for Mixer server.  Proxy is using grpc Channel, could  not use Envoy cluster manager. 

I think we can specify DNS to grpc channel. it should do DNS lookup.